runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1639,6 +1639,16 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 		extensions->_numaManager.shouldEnablePhysicalNUMA(false);
 		extensions->numaForced = true;		
 #endif /* defined(J9VM_GC_VLHGC) || defined(J9VM_GC_GENERATIONAL) */
+	}
+
+	index = option_set(vm, "-XX:disableCPUAffinity", EXACT_MATCH);
```

Comment:
Rather than reparsing the option here, can this code check the thread lib setting?